docs/docs/api-proxy.md

```diff
@@ -56,7 +56,7 @@ module.exports = {
 
 Keep in mind that middleware only has effect in development (with `gatsby develop`).
 
-### Self-signed certificates
+## Self-signed certificates
```

Comment:
It seems like this is sub-section of "Advanced proxying" so h3 feels to me like proper level?